sdk/include/platform/ibex/platform-hardware_revoker.hh

```diff
@@ -24,7 +24,7 @@ namespace Ibex
 {
 	class HardwareRevoker
 	{
-		private:
+		public:
```

Comment:
I don’t like making these fields public just for a benchmark. What API are we missing?

Reply:
Good call.  Have a hopefully much better spin.